osu.Game.Rulesets.Osu/UI/Cursor/GameplayCursor.cs

```diff
@@ -112,7 +112,7 @@ protected override void SkinChanged(ISkinSource skin, bool allowFallback)
             }
 
             [BackgroundDependencyLoader]
-            private void load(OsuConfigManager config, IBindableBeatmap beatmap)
+            private void load(OsuConfigManager config, IBindable<WorkingBeatmap> beatmap)
```

Comment:
Do we really need to remove IBindableBeatmap? It's used as a marker for the OsuGame specific beatmap.

Reply:
This wasn't mentioned anywhere, nor did it affect behaviour. I guess if this is the case we should probably rename it and document it. Personally I couldn't see the point of having it.

Reply:
It was mentioned in the xmldoc of IBindableBeatmap?